### PR TITLE
Always output a packed type for q/dq

### PR DESCRIPTION
**Pull request by pfultz2**

### CODE REVIEW

File: src/include/migraphx/op/dequantizelinear.hpp [hunk 1]
```diff
@@ -54,7 +54,7 @@ struct dequantizelinear
         {
             MIGRAPHX_THROW("DEQUANTIZELINEAR: Zero point and input should be the same type.");
         }
-        return {inputs[1].type(), inputs[0].lens(), inputs[0].strides()};
+        return inputs[0].with_lens(inputs[1].type(), inputs[0].lens());
```
Comment (lakhinderwalia) [inline src/include/migraphx/op/dequantizelinear.hpp:57]: Wouldn't `inputs[0].with_type(inputs[1].type())` do?

Comment (TedThemistokleous) [reply src/include/migraphx/op/dequantizelinear.hpp:57]: I think he needs to set things based on output type as the dequantized output type is based on the scale type used.

**Review by TedThemistokleous — APPROVED** [review]

File: src/include/migraphx/op/dequantizelinear.hpp (hunk 1 above)
Comment (lakhinderwalia) [reply src/include/migraphx/op/dequantizelinear.hpp:57]: The suggested simpler code is doing just that. Use scale type. Thanks.

Comment (pfultz2) [reply src/include/migraphx/op/dequantizelinear.hpp:57]: > Wouldn't inputs[0].with_type(inputs[1].type()) do?

That only changes the type, it doesnt change the strides to make them packed.

Comment (lakhinderwalia) [reply src/include/migraphx/op/dequantizelinear.hpp:57]: Shouldn't the api `with_type` be fixed for this? All that is expected is the `type` to change, nothing else. Thanks.

Comment (lakhinderwalia) [reply src/include/migraphx/op/dequantizelinear.hpp:57]: I meant to fix the `with_type()` for its default behavior. Anyways, it is a discussion for some other time. Thanks.

**Review by lakhinderwalia — APPROVED** [review]

**Pull request merged**